io/eolearn/tests/test_raster_io.py

```diff
@@ -360,3 +360,33 @@ def test_time_dependent_feature_with_timestamps(test_eopatch):
     new_eopatch = import_task(test_eopatch, filename=filename)
 
     assert_array_equal(new_eopatch[feature], test_eopatch[feature])
+
+
+@pytest.mark.parametrize(
+    "no_data_value, data_type", [(None, np.uint16), (np.nan, float), (0, int), (None, float), (1, np.byte)]
+)
+def test_export_import_sequence(no_data_value, data_type):
```

Comment:
a quick docstring describing the tests would be nice. Doesn't have to be much, just to guide whoever tries to adjust the tests.